lib/tty.js

```diff
@@ -48,6 +48,13 @@ function WriteStream(fd) {
     writable: true
   });
 
+  // Prevents interleaved stdout/stderr output in OS X terminals.
```

Comment:
I don't think this comment is correct. The change is intended to fix loosing output when doing process.exit. Interleaving will depend on OS level buffering. Here is the full backstory in case you want to link to it: https://github.com/nodejs/node/issues/6806#issuecomment-222350923